laws/src/main/scala/algebra/laws/JoinSemilatticeLaws.scala

```diff
@@ -0,0 +1,13 @@
+package algebra.laws
+
+import algebra.lattice.JoinSemilattice
+import cats.kernel.laws.SemilatticeLaws
+
+trait JoinSemilatticeLaws[A] extends SemilatticeLaws[A] {
```

Comment:
A `JoinSemilattice` does not extend `Semilattice`, so why should the laws follow the same pattern?